client/starwhale/api/_impl/model.py

```diff
@@ -21,6 +22,8 @@ def build(
     name: t.Optional[str] = None,
     project_uri: str = "",
     desc: str = "",
+    remote_instance_uri: t.Optional[str] = None,
+    remote_project: t.Optional[str] = None,
```

Comment:
does remote_project uri have already include the remote remote_instance_uri info? If we use the URI format.

Reply:
fixed